examples/using/using.ino

```diff
@@ -0,0 +1,30 @@
+/*
+  This example demonstrate pH value reading
+*/
+#include "Arduino.h"
+
+// Include library
+#include "troykaph.h"
+
+TroykaPH phMeter(A4); // set used analog pin
+
+uint32_t showingTime;
+
+void setup() {
+    phMeter.begin(); // if module not calibrated or...
+    // phMeter.begin(correction,zeroShift); // if you have it (use calibrate.ino) for it
+
+    Serial.begin(9600);
+    showingTime = millis() + 3000; // show result once per 3 seconds
+}
+
+void loop() {
+    phMeter.update(1000); // real read from sensor once per second
+    // (you can increase this period, in practice pH value changing too slowly)
+
+    if (showingTime <= millis()) {
```

Comment:
I think there might be problems here when the `showingTime` variable overflows. It will become close to zero, and the `millis()` will still be around 2^32.  It is much better to use something like this:
```suggestion
    if (millis() - showingTime > interval) {
```

Reply:
hard to explain. It all depends on what we consider the problem. In both cases, at the moment of crossing zero, update will run twice without a pause. But your version is clearer, therefore I will redo it.

Reply:
Done